wiki/templates/base.html

```diff
@@ -2,10 +2,22 @@
 <html class="no-js" lang="en">
     <head>
         <meta charset="utf-8">
-        <title>
-			{% if pagetitle is defined %}{{ pagetitle }} | {% endif %}{% if post is defined and post.title is defined %}{{ post.title }} | {% endif %}Propolis - web3 Wiki
-		</title>
-        <meta name="description" content="">
+
+		{% set pagetitle = pagetitle + ' | Propolis - web3 Wiki' %}
+		{% set pagedescription = "Propolis is a wiki which utilizes the Hive blockchain for data storage." %}
```

Comment:
I would like to use the post.body here instead of static string. A strip_markdown() function is needed to that, otherwise Markdown formatting characters will be present in the string, which is not desired. This description must be plain text. The ToastUI js handles the Markdown rendering in the browser, but this string needs to be part of the static HTML content for content previews to work correctly.